### DISCUSSION TITLE
Vue Firebase Authentication/Comment Section Not Working

### DISCUSSION THREAD

**Discussion by elle-concetta** [vuejs/vuefire#1547] [Questions and Help] [answered]
Hello everyone!

So I've created a wedding web app with vue/vite and firebase. The only thing that won't work is the authentication/comment section. Can anyone tell me what I am doing wrong?

Here is the link to my authcomments.vue file: https://github.com/elle-concetta/wedding-application/blob/main/src/components/TheAuthComments.vue

Here is the link to the repository: https://github.com/elle-concetta/wedding-application

Here is a short clip of what is happening in the auth/comment section: 
<video src='https://github.com/vuejs/vuefire/assets/128157075/4ff3385c-c34e-48a4-84ed-a7108ebbf9a6' width=180/>

**Comment by posva** [comment, marked as answer]
It seems like the popup is being blocked by the browser, you need to allow it. You can also use signinWithRedirect

**Reply by elle-concetta** [reply]
Thank you so much! You are a gem 💎💎
